native_client/dotnet/DeepSpeechClient/Structs/ModelState.cs

```diff
@@ -7,6 +7,8 @@
 
 namespace DeepSpeechClient.Structs
 {
+    //FIXME: ModelState is an opaque pointer to the API, why is this code reverse
+    // engineering its contents?
     [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Ansi)]
```

Comment:
I see. I think IntPtrs would be safer as this struct is likely to get outdated and then someone using it would get memory corruption and all kinds of nastiness. I'll do that in a follow-up PR. Thanks for the review!